Formula/s/solo2-cli.rb

```diff
@@ -0,0 +1,30 @@
+class Solo2Cli < Formula
+  desc "CLI to update and use Solo 2 security keys"
+  homepage "https://solokeys.com/"
+  url "https://github.com/solokeys/solo2-cli/archive/refs/tags/v0.2.2.tar.gz"
+  sha256 "49a30c5ee6f38be968a520089741f8b936099611e98e6bf2b25d05e5e9335fb4"
+  license any_of: ["Apache-2.0", "MIT"]
+  head "https://github.com/solokeys/solo2-cli.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "libusb"
```

Comment:
There is no linkage against `libusb`. Is it really used?

Reply:
the hidapi crate either needs libusb or hidraw, see https://docs.rs/hidapi/latest/hidapi/#linux-backends

Reply:
see also the comment in the AUR https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=solo2-cli

Reply:
But it's statically linking against hidraw, which means libusb is not used: https://github.com/solokeys/solo2-cli/blob/v0.2.2/Cargo.toml#L26

```toml
hidapi = { version = "2", default-features = false, features = ["linux-static-hidraw"] }
```

Which should also be the reason why it has linkage against libudev, according to https://github.com/ruabmbua/hidapi-rs/blob/b69e5db0b7267901da65910d563e9d142e491948/build.rs#L58.